api-common-java/.github/workflows/downstream.yaml

```diff
@@ -1,144 +0,0 @@
-        - area120-tables
-        - artifact-registry
-        - asset
-        - assured-workloads
-        - automl
-        - bigquery
-        - bigqueryconnection
-        - bigquerydatatransfer
-        - bigquerymigration
-        - bigqueryreservation
-        - bigtable
-        - billing
-        - billingbudgets
-        - binary-authorization
-        - channel
-        - cloudbuild
-        - compute
-        - contact-center-insights
-        - container
-        - containeranalysis
-        - data-fusion
-        - datacatalog
-        - dataflow
-        - datalabeling
-        - dataproc
-        - dataproc-metastore
-        - datastore
-        - datastream
-        - debugger-client
-        - deploy
-        - dialogflow
-        - dialogflow-cx
-        - dlp
-        - dms
-        - dns
-        - document-ai
-        - domains
-        - errorreporting
-        - essential-contacts
-        - eventarc
-        - filestore
-        - firestore
-        - functions
-        - game-servers
-        - gke-connect-gateway
-        - gkehub
-        - gsuite-addons
-        - iam-admin
-        - iamcredentials
-        - iot
-        - kms
-        - language
-        - life-sciences
-        - logging
-        - logging-logback
-        - managed-identities
-        - mediatranslation
-        - memcache
-        - monitoring
-        - monitoring-dashboards
-        - network-management
-        - network-security
-        - networkconnectivity
-        - notebooks
-        - orchestration-airflow
-        - orgpolicy
-        - os-config
-        - os-login
-        - phishingprotection
-        - policy-troubleshooter
-        - private-catalog
-        - profiler
-        - pubsublite
-        - recaptchaenterprise
-        - recommendations-ai
-        - recommender
-        - redis
-        - resource-settings
-        - resourcemanager
-        - retail
-        - scheduler
-        - secretmanager
-        - security-private-ca
-        - securitycenter
-        - securitycenter-settings
-        - service-control
-        - service-management
-        - service-usage
-        - servicedirectory
-        - shell
-        - spanner
-        - spanner-jdbc
-        - speech
-        - storage
-        - storage-nio
-        - storage-transfer
-        - talent
-        - tasks
-        - texttospeech
-        - tpu
-        - trace
-        - translate
-        - video-intelligence
-        - video-transcoder
-        - vision
-        - vpcaccess
-        - webrisk
-        - websecurityscanner
-        - workflow-executions
-        - workflows
-    steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-java@v3
-      with:
-        distribution: temurin
-        java-version: ${{matrix.java}}
-    - run: java -version
-    - run: sudo apt-get update -y
-    - run: sudo apt-get install libxml2-utils
-    - run: .kokoro/downstream-client-library-check.sh api-common ${{matrix.repo}}
```

Comment:
Are these downstream checks replaced with anything, or have we decided to remove them?

Reply:
I have no idea about this.

Reply:
I don't think we have a equivalent yet, same thing for `downstream.yml` in gax-java module. They are valuable to verify that every change to `api-common-java` and `gax-java` are still compatible with all downstream libraries. They were never required checks so I think it's OK to merge this PR in without an equivalent but we should create an issue in the backlog to keep track of it.